File: src/sdk-reference/cpp/1/collection/update-mapping/snippets/update-mapping.cpp
try {
  std::string mapping = R"({
    "dynamic": "false",
    "_meta": {
      "area": "Panipokhari"
    },
    "properties": {
      "plate": { "type": "keyword" }
    }
  })";

  kuzzle->collection->updateMapping("nyc-open-data", "yellow-taxi", mapping);

  std::cout << "Mapping successfully updated" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/collection/update-mapping/snippets/update-mapping.cs
try {
  string mapping = @"{
    ""properties"": {
      ""plate"": { ""type"": ""keyword"" }
    }
  }";

  kuzzle.collection.updateMapping("nyc-open-data", "yellow-taxi", mapping);

  Console.WriteLine("Mapping successfully updated");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
